### ISSUE TITLE
Question - What are you doing in place?

### ISSUE THREAD

**Issue by karl0ss** [greghesp/assistant-relay#253]
I noticed on your readme that you say that you have fully migrated to HA, so you don't need this now....I am acutally using this in conjunction with HA to be able to "broadcast" around the house, using the correct way, (that doesn't kill playing music and such)..

I am wondering what you are now doing/using if you are no longer using this approach? as this still seems to be the best to me...

**Comment by Madj42** [comment]
I tried asking the same thing on Twitter and haven't heard anything.  I'm curious too.

**Comment by rlatimer4** [comment]
I would also like to know.

**Comment by jherby2k** [comment]
@greghesp I'm also an HA user who is curious how you're approaching this now. I use this feature quite a bit for broadcasting announcements (green house is too hot, washing machine is done etc) and don't want to use HA's TTL service because it interrupts music that might be playing. Happy to drop this project if there's a better way!

**Comment by JOHLC** [comment]
Also interested if anyone finds something

**Comment by leocaseiro** [comment]
Has anyone tried this approach?

https://community.home-assistant.io/t/send-commands-actions-to-google-home-with-buttons-on-ha-frontend/98532/9


I just found it, will try this week.

**Comment by jherby2k** [comment]
Try this: https://community.home-assistant.io/t/script-to-resume-google-cast-devices-after-they-have-been-interrupted-by-any-action/383896

Works great!

**Comment by JumpMaster** [comment]
Hi all,

Starting with Home Assistant 2023.1 you are able to use the Google Assistant SDK directly from Home Assistant.  I'm currently using the beta and I'm able to broadcast from Node-Red using the Home Assistant service.

https://rc.home-assistant.io/integrations/google_assistant_sdk/

At last I'm able to shutdown this container.  Thanks for your hard work @greghesp

**Comment by greghesp** [comment]
> Hi all,
> 
> Starting with Home Assistant 2023.1 you are able to use the Google Assistant SDK directly from Home Assistant. I'm currently using the beta and I'm able to broadcast from Node-Red using the Home Assistant service.
> 
> https://rc.home-assistant.io/integrations/google_assistant_sdk/
> 
> At last I'm able to shutdown this container. Thanks for your hard work @greghesp

Ahh fantastic news. I thought this was a WIP as spotted it the code.

**Comment by Madj42** [comment]
Can confirm that I was able to do the same a few days ago after switching to the beta.  Was waiting a few days in order to suggest it as a replacement.  So far I'm extremely happy.

**Comment by karl0ss** [comment]
Yep, seems to work nicely in HA, no need for this, or the arm docker image I built anymore... :)

**Comment by vanyacooper** [comment]
I confirm I have installed and configured this with the 2023.1 version and it is working great. Thanks again to everyone that contributed to this project!

I just wanted to mention that I also found another version of a similar project that I never got around to installing or testing at https://github.com/AndBobsYourUncle/hassio-addons/tree/master/google-assistant-webserver if anyone is interested in checking it out as a programming example. It seems to be somewhat simpler and written in Python but I will probably never get around to it now that there is a built in option. And this is not any kind of a recommendation or endorsement since I haven't actually tried it.
